search/main.py

```diff
@@ -23,11 +23,14 @@ def __init__(self):
         # Data available to the dropdown React app in docs, used to fill out autocomplete results.
         self.displayed_attributes = ['title', 'text', 'url', 'site', 'section']
         # Data actually searchable by our queries.
-        self.searchable_attributes = ['title', 'pageUrl', 'text', 'url', 'section']
+        self.searchable_attributes = ['title', 'pageUrl', 'section', 'url', 'text']
 
         # Show results for one query with the listed pages, when they by default would not show up as best results.
         # Note: these aren't automatically two-way, which is why they're all defined twice.
         self.synonyms = {
+            "cron": ["crons"],
+            "crons": ["cron tasks", "cron jobs"],
+            "e-mail": ["email"],
```

Comment:
I noticed that for the following, the first search result is suboptimal:
- "hooks"  should probably bring up-> `/configuration/app/hooks.md`
- "deploy" -> `/configuration/app/hooks.html#deploy-hook`

Reply:
Given that the pages come up in the top 2 or 3 results, I'm not sure I'd try to change too much more to fix that right now. I'm not sure how to optimize for that without affecting other results. Can look into it again soon, though. Should be re-evaluating search anyway.